docs/source/index.rst

```diff
@@ -28,6 +28,7 @@ modularity and versatility for a broad set of industry use cases.
    developapps/developing_applications
    tutorials
    ops_guide
+	 upgrade_to_newest_version
```

Comment:
I'm not seeing the Upgrade doc in the left toc. Can you check we have this right? Could the indentation be off?
![Screen Shot 2019-12-02 at 5 17 55 PM](https://user-images.githubusercontent.com/36892852/69999877-b868b880-1527-11ea-8f13-f1094672fb5d.png)

Reply:
This seems to be fixed now. When DocBuild starts running successfully we can check the build.